app/routes/game.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  // TODO: insert yourself into the watchers array unless you're already in one
```

Comment:
Do you think people should have to be logged in in order to watch? I guess I'm not opposed to requiring authentication to participate since they'll be contributing to the firebase bill :)